tests/parsec/synonyms/bin/synonyms.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python3
```

Comment:
Looks removing the shebang in test files causes the functional tests to fail.

All 4 functional test builds failed in Travis @sgaist. Unfortunately they timed out, so there is no helpful output there (:angry:). So I've executed one that failed locally (`./tests/parsec/synonyms/05-float_list.t`) and it seems to have failed as the shell script will try to actually run the python script. See screenshot of the test:

![image](https://user-images.githubusercontent.com/304786/65954039-bfc90280-e4a1-11e9-99ab-bb28cd0eaf9f.png)

Could you revert the change for the `./tests/` directory, please @sgaist ? Thanks!

Reply:
Sure thing